authorization-service/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "authorization-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "deploy": "npx sls deploy",
+    "deploy:importProductsFile": "npx sls deploy --function importProductsFile",
```

Comment:
does this two scripts related to authorization-service?

Reply:
Yes, it was copypaste from import service
While AuthorizarionService consist of only one lambda, was using only deploy script and didn't notice this error, thanks